app/src/main/java/protect/card_locker/preferences/SettingsActivity.java

```diff
@@ -187,6 +187,17 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
                 return true;
             });
 
+            // Add the "Use volume keys to switch visible card" option
+            Preference volumeKeyPreference = findPreference(getResources().getString(R.string.settings_key_use_volume_keys));
+            if (volumeKeyPreference != null) {
+                volumeKeyPreference.setOnPreferenceChangeListener((preference, newValue) -> {
+                    // Handle enabling/disabling the volume keys for navigation
+                    boolean useVolumeKeys = (Boolean) newValue;
+                    // You can add additional logic here if necessary
+                    return true;  // Save the new value
+                });
+            }
+
```

Comment:
I didn't test it but are you sure this is necessary? The `screen on` setting also doesn't have any special code here. I think that's just handled correctly by the SettingsActivity by default.

Reply:
Forgot to remove this code, thanks for the note.